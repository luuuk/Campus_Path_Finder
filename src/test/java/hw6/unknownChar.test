# This test finds a path from a node to a node not in graph

LoadGraph test test.tsv

FindPath test a g

FindPath test r a

FindPath test q w