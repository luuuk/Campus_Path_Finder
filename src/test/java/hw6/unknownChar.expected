# This test finds a path from a node to a node not in graph

loaded graph test

unknown character g

unknown character r

unknown character q
unknown character w